tests/data_association/test_point3d_initializer.py

```diff
@@ -13,6 +13,7 @@
 import numpy as np
 from gtsam import Cal3_S2, Cal3Bundler, PinholeCameraCal3Bundler, Point2, Point3, Pose3, Rot3
 from gtsam.examples import SFMdata
+from numpy.lib.twodim_base import tri
```

Comment:
thanks for the updates @travisdriver. what is this import for?

Reply:
Not sure, I didn't add anything to the unit tests file